owners/test/owners_bot.test.js

```diff
@@ -220,6 +219,32 @@ describe('owners bot', () => {
         expect(true).toBe(true);
       });
     });
+
+    it('does not create review requests', async done => {
+      await ownersBot.runOwnersCheck(github, pr);
+
+      sandbox.assert.notCalled(github.createReviewRequests);
+      done();
+    });
+
+    describe('when the PR description contains #addowners', () => {
+      beforeEach(() => {
+        pr.description = 'Assign reviewers please #addowners';
```

Comment:
Might be worth adding a test where the description contains embedded HTML and / or code tags.

Reply:
Discussed offline: The worst case is not a particularly bad case (reviewers get assigned when we don't particularly expect them to), and even that is rare.